apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/derived/DerivedTableInterface.java

```diff
@@ -16,7 +16,7 @@
  *
  * @author yuriy.larin
  */
-public interface DerivedTableInterface<T extends DerivedEntity> {
+public interface DerivedTableInterface<T extends DerivedEntity> extends Searchable {
```

Comment:
DerivedTable is not searchable by default

Reply:
Removed